.github/workflows/gradle.yml

```diff
@@ -11,12 +11,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
+        java: [17]
```

Comment:
```suggestion
        java: [17, 21]
```

Reply:
@jo-elimu This will be done in another PR here https://github.com/elimu-ai/launcher/pull/76, since Java 21 is only supported from 8.5 onwards.